diff -r 53e01f4f2e9d CMake/ConfigureBuild.cmake
--- a/CMake/ConfigureBuild.cmake	Sat Jun 08 15:53:17 2019 -0300
+++ b/CMake/ConfigureBuild.cmake	Wed Jul 31 08:36:01 2019 +0000
@@ -327,19 +327,17 @@
 endforeach(flag_var)
 endif(OGRE_CONFIG_STATIC_LINK_CRT)
 
-### Commented because the FindOGRE script can currently fill this role better ###
-# # Create the CMake package files
-# if (WIN32)
-#   set(OGRE_CMAKE_DIR CMake)
-# elseif (UNIX)
-#   set(OGRE_CMAKE_DIR lib/cmake)
-# elseif (APPLE)
-# endif ()
-# configure_file(${OGRE_TEMPLATES_DIR}/OGREConfig.cmake.in ${OGRE_BINARY_DIR}/cmake/OGREConfig.cmake @ONLY)
-# configure_file(${OGRE_TEMPLATES_DIR}/OGREConfigVersion.cmake.in ${OGRE_BINARY_DIR}/cmake/OGREConfigVersion.cmake @ONLY)
-# install(FILES
-#   ${OGRE_BINARY_DIR}/cmake/OGREConfig.cmake
-#   ${OGRE_BINARY_DIR}/cmake/OGREConfigVersion.cmake
-#   DESTINATION ${OGRE_CMAKE_DIR}
-# )
-#
+# Create the CMake package files
+if (WIN32)
+  set(OGRE_CMAKE_DIR CMake)
+elseif (UNIX)
+  set(OGRE_CMAKE_DIR lib/OGRE/cmake)
+elseif (APPLE)
+endif ()
+configure_file(${OGRE_TEMPLATES_DIR}/OGREConfig.cmake.in ${OGRE_BINARY_DIR}/cmake/OGREConfig.cmake @ONLY)
+configure_file(${OGRE_TEMPLATES_DIR}/OGREConfigVersion.cmake.in ${OGRE_BINARY_DIR}/cmake/OGREConfigVersion.cmake @ONLY)
+install(FILES
+   ${OGRE_BINARY_DIR}/cmake/OGREConfig.cmake
+   ${OGRE_BINARY_DIR}/cmake/OGREConfigVersion.cmake
+   DESTINATION ${OGRE_CMAKE_DIR}
+)
