diff -r 53e01f4f2e9d CMake/Dependencies.cmake
--- a/CMake/Dependencies.cmake	Sat Jun 08 15:53:17 2019 -0300
+++ b/CMake/Dependencies.cmake	Wed Jul 10 17:59:11 2019 +0000
@@ -176,18 +176,6 @@
   macro_log_feature(Cg_FOUND "cg" "C for graphics shader language" "http://developer.nvidia.com/object/cg_toolkit.html" FALSE "" "")
 endif ()
 
-# Find Boost
-# Prefer static linking in all cases
-if (WIN32 OR APPLE)
-	set(Boost_USE_STATIC_LIBS TRUE)
-else ()
-	# Statically linking boost to a dynamic Ogre build doesn't work on Linux 64bit
-	set(Boost_USE_STATIC_LIBS ${OGRE_STATIC})
-endif ()
-if (APPLE AND OGRE_BUILD_PLATFORM_APPLE_IOS)
-    set(Boost_USE_MULTITHREADED OFF)
-endif()
-
 if(ANDROID)
     # FindBoost needs extra hint on android 
     set(Boost_COMPILER -gcc)
@@ -196,16 +184,6 @@
 set(Boost_ADDITIONAL_VERSIONS "1.57" "1.57.0" "1.56" "1.56.0" "1.55" "1.55.0" "1.54" "1.54.0" "1.53" "1.53.0" "1.52" "1.52.0" "1.51" "1.51.0" "1.50" "1.50.0" "1.49" "1.49.0" "1.48" "1.48.0" "1.47" "1.47.0" "1.46" "1.46.0" "1.45" "1.45.0" "1.44" "1.44.0" "1.42" "1.42.0" "1.41.0" "1.41" "1.40.0" "1.40")
 # Components that need linking (NB does not include header-only components like bind)
 set(OGRE_BOOST_COMPONENTS thread date_time)
-find_package(Boost COMPONENTS ${OGRE_BOOST_COMPONENTS} QUIET)
-if (NOT Boost_FOUND)
-	# Try again with the other type of libs
-	if(Boost_USE_STATIC_LIBS)
-		set(Boost_USE_STATIC_LIBS OFF)
-	else()
-		set(Boost_USE_STATIC_LIBS ON)
-	endif()
-	find_package(Boost COMPONENTS ${OGRE_BOOST_COMPONENTS} QUIET)
-endif()
 
 if(Boost_FOUND AND Boost_VERSION GREATER 104900)
     if(Boost_VERSION GREATER 105300)
